string current_behaviour
string current_task
